language/functional_tests/src/evaluator.rs

```diff
@@ -238,7 +238,9 @@ fn get_transaction_parameters<'a>(
             )
         }),
         gas_unit_price: 1,
-        expiration_time: Duration::from_secs(u64::max_value()),
+        expiration_time: config
+            .expiration_time
+            .unwrap_or_else(|| Duration::from_secs(u64::max_value()/1_000_000)),
```

Comment:
Will this default be a problem once we check that the expiration date is not too far in the future?

Reply:
I'm just about to update this!